src/components/ContentEditable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+
+const DEFAULT_CLASSES = "focus:outline-none"
+
+export default function ContentEditable({
+  value,
+  onSubmit,
+  className = "",
+  nextFocus = null,
+}) {
+  const [content, setContent] = React.useState(value);
+  
+  const handleChange = evt => {
+    setContent(evt.target.innerHTML);
+  }
+  
+  const handleSubmit = () => {
+    onSubmit(content);
+  };
+  
+  const onKeyDown = (evt) => {
+    if ((evt.metaKey && evt.code == "KeyS") || evt.key === "Enter") {
+      evt.preventDefault();
+      onSubmit(content);
+      if (nextFocus) {
+        nextFocus();
+      }
+    }
+  }
+  
+  return (
+    <div
+      className={`${DEFAULT_CLASSES} ${className}`}
+      contentEditable={true}
+      suppressContentEditableWarning
```

Comment:
React has mixed feelings about contenteditable, and will emit a warning when you use it. 

Given that we parse input on save (which happens on blur, enter, cmd + s) I believe this should be fine in the same way that Quill is fine.